src/main/java/org/folio/auth/authtokenmodule/storage/RefreshTokenStore.java

```diff
@@ -62,12 +62,15 @@ public RefreshTokenStore(Vertx vertx, String tenant) {
 
   /**
    * Save the token to the token store. This should be done anytime a new refresh token
-   * is issued.
+   * is issued. Also cleans up any expired tokens when called.
    * @param refreshToken The refresh token to store.
+   * @param waitForCleanupExpired True if the caller should wait until the cleanup of expired
+   * tokens succeeds, otherwise return after the save is complete and clean up the expired
+   * tokens in the background.
    * @return A failed future should the save operation fail. Otherwise a succeeded future
    * is returned.
```

Comment:
The completion of the save, or the completion of both the save and the cleanup if waitForCleanupExpired is true.